puppet/modules/quickstack/manifests/neutron/compute.pp

```diff
@@ -127,6 +129,20 @@
       vxlan_udp_port      => $ovs_vxlan_udp_port,
       veth_mtu            => $veth_mtu,
     }
+
+    # check if opendaylight needs to be configured.
+    if ('opendaylight' in $ml2_mechanism_drivers) {
+      # OVS manager
+      exec { 'Set OVS Manager to OpenDaylight':
+        command => "/usr/bin/ovs-vsctl set-manager tcp:${odl_controller_ip}:6640",
+        unless  => "/usr/bin/ovs-vsctl show | /usr/bin/grep 'Manager \"tcp:${odl_controller_ip}:6640\"'",
```

Comment:
neutron/controller is neutron connecting to odl,
this one is ovs connecting to odl, they connect on separate ports